extensions/wikia/PageHeaderExperiments/PageHeaderExperiments.hooks.php

```diff
@@ -0,0 +1,9 @@
+<?php
+class PageHeaderExperimentsHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)